### DISCUSSION TITLE
[Spec] Prioritize integration with Azure Communication Services

### DISCUSSION THREAD

**Discussion by MhAllan** [dotnet/maui#327] [Ideas]
Azure is getting communication services ready soon, which is a bit late for what people and market want.

Having that, we hope MAUI will come fully supporting, especially the video calls.

**Comment by nbevans** [comment]
Surely that would be the job of ACS to provide a library for MAUI? Why would MAUI core team want to maintain that?

**Comment by MhAllan** [comment]
Because ACS can't do anything if you don't give them camera and mic and live video player?

**Comment by Sarthak261995** [comment]
Check this https://github.com/Laerdal/Xamarin.AzureCommunicationCalling
